react/e2e/private/historyScreen/inventory.test.js

```diff
@@ -14,4 +14,12 @@ describe('historyScreen inventory', () => {
     )
     expect(currentAccountBalanceIndicator).toEqual(1)
   })
+
+  it('displays 20 most recent historyItemIndicator in descending time', async () => {
```

Comment:
"20" and "most recent" not tested here

please change title to describe current test